### ISSUE TITLE
Documentation Notebooks

### ISSUE THREAD

**Issue by bhargavvader** [RaRe-Technologies/gensim#3039]
Hello, I was going through some documentation notebooks, and noticed that many of them ([Poincare Embeddings](https://github.com/RaRe-Technologies/gensim/blob/develop/docs/notebooks/Poincare%20Tutorial.ipynb), [WikiNews](https://github.com/RaRe-Technologies/gensim/blob/develop/docs/notebooks/wikinews-bigram-en.ipynb), [Varembed](https://github.com/RaRe-Technologies/gensim/blob/develop/docs/notebooks/Varembed.ipynb)) have been uploaded without being run to the end, they fail with errors halfway through.

@piskvorky (and others), would it be useful to have these updated? Or is someone from the RaRe / gensim team in charge of documentation?

**Comment by bhargavvader** [comment]
Also, if there are any modules (embeddings or topic models) with no documentation or examples of usage, I am happy to help make them.

**Comment by mpenkov** [comment]
I've been trying to steer documentation away from notebooks and towards executable python scripts for this very reason. Unless somebody regularly re-runs the notebooks (and checks that they output the right result), they become significantly less useful.

If you can port the notebook to a reproducible example in the [gallery](https://github.com/RaRe-Technologies/gensim/tree/develop/docs/src/gallery) then that would be the best way forward. If for whatever reason the notebook has to remain a notebook, then sure, update it, but at some point down the line it will go out of date and people will raise a ticket similar to this one.

**Comment by bhargavvader** [comment]
Fair enough. For porting it to a reproducible example, is it pretty much changing the code to a  `.py` file, with the text commented? How exactly would one go about it?

**Comment by mpenkov** [comment]
Please see the [documentation](https://radimrehurek.com/gensim/auto_examples/howtos/run_doc.html#sphx-glr-auto-examples-howtos-run-doc-py).

**Comment by piskvorky** [comment]
That `helpful script` link in that documentation is broken (404).

As part of your PR, could you please change it to https://github.com/mpenkov/gensim/blob/develop/docs/src/tools/to_python.py ? Thanks!
